File: articles/foresight-data-systems-and-reproducible-workflows/c/data_quality_score.c
#include <stdio.h>
double data_quality(double c,double v,double f,double t){return 0.25*c+0.25*v+0.25*f+0.25*t;}
int main(void){printf("Public trust driver data quality: %.4f\n", data_quality(1.0,0.95,0.78,0.70)); return 0;}

File: articles/foresight-data-systems-and-reproducible-workflows/fortran/data_quality_score.f90
program data_quality_score
  implicit none
  real :: score
  score = 0.25*1.00 + 0.25*0.95 + 0.25*0.78 + 0.25*0.70
  print *, "Public trust driver data quality:", score
end program data_quality_score
